sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,196 @@ def __repr__(self):
+    >>> loads = friction.to_loads() + spring.to_loads()
+    >>> fr, frstar = kane.kanes_equations([block], loads)
+    >>> eom = sm.Matrix([fr, frstar])
+
+    Parameters
+    ==========
+
+    f_n : Expr
+        The normal force between the surfaces. It should always be a non-negative scalar.
+    mu_k : Expr
+        The coefficient of kinetic friction.
+    pathway : PathwayBase
+        The pathway that the actuator follows.
+    v_s : Expr
+        The Stribeck friction coefficient.
+    sigma : Expr
+        The viscous friction coefficient.
+    mu_s : Expr, optional
+        The coefficient of static friction. Defaults to mu_k, meaning the Stribeck effect evaluates to 0 by default.
+
+    References
+    ==========
+
+    .. [Moore2022] https://moorepants.github.io/learn-multibody-dynamics/loads.html#friction.
+    .. [Flores2023] Paulo Flores, Jorge Ambrosio, Hamid M. Lankarani,
+            "Contact-impact events with friction in multibody dynamics: Back to basics",
+            Mechanism and Machine Theory, vol. 184, 2023. https://doi.org/10.1016/j.mechmachtheory.2023.105305.
+    .. [Rogner2017] I. Rogner, "Friction modelling for robotic applications with planar motion",
+            Chalmers University of Technology, Department of Electrical Engineering, 2017.
+
+    """
+
+    def __init__(self, mu_k, f_n, pathway, *, v_s=None, sigma=0, mu_s=None):
+        self.mu_k = mu_k if mu_k is not None else 1
+        self.mu_s = mu_s if mu_s is not None else self.mu_k
+        self.f_n = f_n
+        self.sigma = sigma if sigma is not None else 0
+        self.v_s = v_s if v_s is not None or v_s == 0 else 0.01
+        self.pathway = pathway
+
+    @property
+    def mu_k(self):
+        """The coefficient of kinetic friction."""
+        return self._mu_k
+
+    @mu_k.setter
+    def mu_k(self, mu_k):
+        if hasattr(self, '_mu_k'):
+            msg = (
+                f'Can\'t set attribute `mu_k` to '
+                f'{repr(mu_k)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._mu_k = sympify(mu_k, strict=True)
+
+    @property
+    def mu_s(self):
+        """The coefficient of static friction."""
+        return self._mu_s
+
+    @mu_s.setter
+    def mu_s(self, mu_s):
+        if hasattr(self, '_mu_s'):
+            msg = (
+                f'Can\'t set attribute `mu_s` to '
+                f'{repr(mu_s)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._mu_s = sympify(mu_s, strict=True)
+
+    @property
+    def f_n(self):
+        """The normal force between the surfaces."""
+        return self._f_n
+
+    @f_n.setter
+    def f_n(self, f_n):
+        if hasattr(self, '_f_n'):
+            msg = (
+                f'Can\'t set attribute `f_n` to '
+                f'{repr(f_n)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._f_n = sympify(f_n, strict=True)
```

Comment:
I think if you simply do not define a setter, Python raises the appropriate error if you try to set that attribute. So these aren't needed.

In your init, you just do `self._f_n = f_n` to set it.

Reply:
> In your init, you just do self._f_n = f_n to set it.

I agree, it is better to use `self._f_n = sympify(f_n, strict=True)` in the `__init__` and remove the setters.